WalletWasabi.Fluent/Screenshot/Capture.cs

```diff
@@ -99,52 +99,52 @@ private static void Save(Control? target, Size size, string path)
 		switch (extension.ToLower())
 		{
 			case ".png":
-			{
-				using var stream = File.Create(path);
-				var pixelSize = new PixelSize((int)size.Width, (int)size.Height);
-				var dpiVector = new Vector(96d, 96d);
-				using var bitmap = new RenderTargetBitmap(pixelSize, dpiVector);
-				target.Measure(size);
-				target.Arrange(new Rect(size));
-				bitmap.Render(target);
-				bitmap.Save(stream);
-				break;
-			}
+				{
+					using var stream = File.Create(path);
+					var pixelSize = new PixelSize((int)size.Width, (int)size.Height);
+					var dpiVector = new Vector(96d, 96d);
+					using var bitmap = new RenderTargetBitmap(pixelSize, dpiVector);
+					target.Measure(size);
+					target.Arrange(new Rect(size));
+					bitmap.Render(target);
+					bitmap.Save(stream);
+					break;
```

Comment:
The break should be outside of the { }

Reply:
Why? Sometimes we use it inside and other times we use it inside. IMO we should unify that and remove those curly brackets, they look weird.

https://github.com/zkSNACKs/WalletWasabi/pull/9849#discussion_r1058170383

Reply:
Yes. If no local variable is created, you can remove the curly brackets. For me, it is unknown to put the break inside. 
https://stackoverflow.com/a/222683/1844347

Reply:
Forget it, keep it as it was. We are getting out of scope and the PR is huge - better to merge quickly.

Reply:
I would do this (removing the curly brackets if possible) in a follow up PR, but for the break it was inside the { } before this PR.